TickSpec/TickSpec.fsproj

```diff
@@ -2,15 +2,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <Import Project="../netfx.props" />
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net452</TargetFrameworks>
+    <TargetFrameworks>$(DotNetStandardVersion);$(DotNetFrameworkVersion)</TargetFrameworks>
     <Title>TickSpec</Title>
-    <Description>
-      TickSpec is a lightweight Behaviour Driven Development (BDD) framework for .Net.
-      Describe behaviour in plain text using the Gherkin business language, i.e. Given,
-      When, Then. Easily execute the behaviour against matching F# 'ticked' methods 
-      (let ``tick method`` () = true) or attributed C# or F# methods.
-    </Description>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
   </PropertyGroup>
+  <ItemGroup>
+      <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="6.0.0" />
```

Comment:
Can you add this only when the target is net framework and not netstandard? I believe, that it makes sense to move it to netstandard 2.1. If there are any build issues then we can resolve them. I believe that there are just some compilation flags that caused compilation of code that should not be compiled for netstandard.

Reply:
i used AsyncInterfaces conditionally for dotnet framework only. You were right the compilation of some parts is flagged to be excluded for netstandard.